### PR TITLE
Us4 delete outivity

### PR DESCRIPTION
**Pull request by BirkBee**
Hey, thank you very much for checking my PR 🙌

### CODE REVIEW

File: components/OutivitiesListItem/index.js [hunk 1]
```diff
@@ -2,23 +2,35 @@ import styled from "styled-components";
 import Link from "next/link";
 import Image from "next/image";
 
-export default function OutivitiesListItem({ outivity }) {
+export default function OutivitiesListItem({ outivity, onDeleteOutivity }) {
```
Comment (willywdev) [inline components/OutivitiesListItem/index.js]: Do you need onDeleteOutivity here?

File: components/OutivityDetail/index.js [hunk 2]
```diff
@@ -1,8 +1,20 @@
 import Head from "next/head";
 import styled from "styled-components";
 import Image from "next/image";
+import { useRouter } from "next/router";
+
+export default function OutivityDetail({ outivity, onDeleteOutivity }) {
+  const router = useRouter();
+  const confirmDelete = () => {
+    const confirmed = window.confirm(
```
Comment (willywdev) [inline components/OutivityDetail/index.js:9]: Very nice to use window.confirm! 🚀

File: components/OutivityDetail/index.js [hunk 3]
```diff
@@ -42,4 +57,21 @@ const StyledCountryName = styled.span`
 const StyledOutivityImage = styled(Image)`
   overflow: clip;
   overflow-clip-margin: content-box;
+  height: auto;
```
Comment (willywdev) [inline components/OutivityDetail/index.js:61]: height="auto" could mess with the pages layout until its fully loaded. consider using the layout="fixed"-prop on the Image itself.

File: components/OutivityDetail/index.js [hunk 4]
```diff
@@ -42,4 +57,21 @@ const StyledCountryName = styled.span`
...
+`;
+
+const StyledDeleteButton = styled.button`
```
Comment (willywdev) [inline components/OutivityDetail/index.js:64]: If you have multiple Styles with the same Usage (e.g. Buttons, Images, ...) consider creating a shared component in the future, so you have the same styling across the board.
This doesn't need to be done right now! Just a suggestion :)

File: pages/_app.js [hunk 5]
```diff
@@ -1,18 +1,24 @@
...
-
 import useLocalStorageState from "use-local-storage-state";
+import { useRouter } from "next/router";
 
 export default function App({ Component, pageProps }) {
   const [outivities, setOutivities] = useLocalStorageState("outivities", {
     defaultValue: initialOutivities,
   });
 
+  const router = useRouter();
```
Comment (willywdev) [inline pages/_app.js]: Do you need useRouter() here?

File: pages/_app.js [hunk 6]
```diff
@@ -1,18 +1,24 @@
 import GlobalStyle from "../styles";
 import { initialOutivities } from "@/lib/data";
 import Layout from "@/components/Layout";
-
 import useLocalStorageState from "use-local-storage-state";
+import { useRouter } from "next/router";
```
Comment (willywdev) [inline pages/_app.js]: Do you need useRouter() here?

**Review by willywdev — CHANGES_REQUESTED** [review]
This looks very good! I like your clean code.
I commented just some small things. 
Keep up the good work! 🚀

File: components/OutivityDetail/index.js (hunk 3 above)
Comment (BirkBee) [reply components/OutivityDetail/index.js:61]: Thank you for the tip 😄

File: components/OutivityDetail/index.js (hunk 4 above)
Comment (BirkBee) [reply components/OutivityDetail/index.js:64]: Thank you very much, that's a  very good suggestion, we'll work on that in the future 🤓 :)

File: pages/_app.js (hunk 5 above)
Comment (BirkBee) [reply pages/_app.js]: Yeah true it's not needed, thank you for helping to clean up 🧹

File: pages/_app.js (hunk 6 above)
Comment (BirkBee) [reply pages/_app.js]: Nope, thanks again 😄  🧹

File: components/OutivityDetail/index.js (hunk 2 above)
Comment (BirkBee) [reply components/OutivityDetail/index.js:9]: Thank you very much 😊 🚀

**Comment by BirkBee** [comment]
Thanks a lot for all of the helpful comments! 🚀 😊

**Review by doemser — APPROVED** [review]
Looks good! Well done! 

I am not a fan of using window.confirm and would always prefer a solid Modal in my Apps style, but this is just taste I guess :P

**Comment by BirkBee** [comment]
Thank you 🚀 
If we should have time left in the second sprint, we will work on a solid Model because we do agree 😄

**Pull request merged**